index.html

```diff
@@ -183,115 +183,115 @@
                 </svg>
               </div>
-            </div>
-            <div class="categories__card js-categories-card">
-              <a href="#" class="categories__link"></a>
+            </li>
+            <li class="categories__card js-categories-card">
+              <a href="#" class="categories__link" aria-label="ver todos los audífonos"></a>
               <div class="categories__img">
-                <img src="image/headphones .png" alt="">
+                <img src="image/headphones .png" alt="Audifonos">
               </div>
               <div class="categories__text">
                 <p>Audífonos</p>
-                <svg width="25" height="27" viewBox="0 0 25 27" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <svg width="25" height="27" viewBox="0 0 25 27" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
                   <path d="M4.89343 13.4746H20.8934" stroke="#363535" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
                   <path d="M14.8934 6.96555L20.8934 13.4746L14.8934 19.9836" stroke="#363535" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
                 </svg>
               </div>
-            </div>
-            <div class="categories__card js-categories-card">
-              <a href="#" class="categories__link"></a>
+            </li>
+            <li class="categories__card js-categories-card">
+              <a href="#" class="categories__link" aria-label="ver todas las lamparas"></a>
               <div class="categories__img">
-                <img src="image/Lamps.png" alt="">
+                <img src="image/Lamps.png" alt="lamparas">
               </div>
               <div class="categories__text">
                 <p>Lámparas</p>
-                <svg width="25" height="27" viewBox="0 0 25 27" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <svg width="25" height="27" viewBox="0 0 25 27" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
                   <path d="M4.89343 13.4746H20.8934" stroke="#363535" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
                   <path d="M14.8934 6.96555L20.8934 13.4746L14.8934 19.9836" stroke="#363535" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
                 </svg>
               </div>
-            </div>
-          </div>
+            </li>
+          </ul>
         </div>
-        <button class="prev__card js-prev-card js-moved-slide" data-direction="-1">&#10094;</button>
-        <button class="next__card js-next-card js-moved-slide" data-direction="1">&#10095;</button>
+        <button class="prev__card js-prev-card js-moved-slide" data-direction="-1" aria-label="izquierda">&#10094;</button>
+        <button class="next__card js-next-card js-moved-slide" data-direction="1" aria-label="derecha">&#10095;</button>
```

Comment:
Bien, pero puede ser mas claro.
```suggestion
        <button class="prev__card js-prev-card js-moved-slide" data-direction="-1" aria-label="navegar a la izquierda">&#10094;</button>
        <button class="next__card js-next-card js-moved-slide" data-direction="1" aria-label="navegar a la derecha">&#10095;</button>
```